diff --git a/.npmrc b/.npmrc
index 50d910c..472b622 100644
--- a/.npmrc
+++ b/.npmrc
@@ -5,2 +5,3 @@ runtime="electron"
 build_from_source="true"
+build_from_source_native_keymap="no"
 legacy-peer-deps="true"
diff --git a/build/.moduleignore b/build/.moduleignore
index ed36151..5b040cc 100644
--- a/build/.moduleignore
+++ b/build/.moduleignore
@@ -65,7 +65,7 @@ fsevents/test/**
 
-native-keymap/binding.gyp
-native-keymap/build/**
-native-keymap/src/**
-native-keymap/deps/**
-!native-keymap/build/Release/*.node
+@vscodium/native-keymap/binding.gyp
+@vscodium/native-keymap/build/**
+@vscodium/native-keymap/src/**
+@vscodium/native-keymap/deps/**
+!@vscodium/native-keymap/build/Release/*.node
 
diff --git a/eslint.config.js b/eslint.config.js
index 47eeebf..f8077c2 100644
--- a/eslint.config.js
+++ b/eslint.config.js
@@ -1465,3 +1465,3 @@ export default tseslint.config(
 						'node:module',
-						'native-keymap',
+						'@vscodium/native-keymap',
 						'native-watchdog',
diff --git a/package.json b/package.json
index efeaec1..b2a6703 100644
--- a/package.json
+++ b/package.json
@@ -108,3 +108,3 @@
     "native-is-elevated": "0.8.0",
-    "native-keymap": "^3.3.5",
+    "@vscodium/native-keymap": "3.3.7-258424",
     "native-watchdog": "^1.4.1",
diff --git a/src/vs/platform/environment/test/node/nativeModules.integrationTest.ts b/src/vs/platform/environment/test/node/nativeModules.integrationTest.ts
index c30c6da..ca6cea2 100644
--- a/src/vs/platform/environment/test/node/nativeModules.integrationTest.ts
+++ b/src/vs/platform/environment/test/node/nativeModules.integrationTest.ts
@@ -44,8 +44,8 @@ flakySuite('Native Modules (all platforms)', () => {
 	test('native-keymap', async () => {
-		const keyMap = await import('native-keymap');
-		assert.ok(typeof keyMap.onDidChangeKeyboardLayout === 'function', testErrorMessage('native-keymap'));
-		assert.ok(typeof keyMap.getCurrentKeyboardLayout === 'function', testErrorMessage('native-keymap'));
+		const keyMap = await import('@vscodium/native-keymap');
+		assert.ok(typeof keyMap.onDidChangeKeyboardLayout === 'function', testErrorMessage('@vscodium/native-keymap'));
+		assert.ok(typeof keyMap.getCurrentKeyboardLayout === 'function', testErrorMessage('@vscodium/native-keymap'));
 
 		const result = keyMap.getCurrentKeyboardLayout();
-		assert.ok(result, testErrorMessage('native-keymap'));
+		assert.ok(result, testErrorMessage('@vscodium/native-keymap'));
 	});
diff --git a/src/vs/platform/keyboardLayout/electron-main/keyboardLayoutMainService.ts b/src/vs/platform/keyboardLayout/electron-main/keyboardLayoutMainService.ts
index 8950ce2..f31cea6 100644
--- a/src/vs/platform/keyboardLayout/electron-main/keyboardLayoutMainService.ts
+++ b/src/vs/platform/keyboardLayout/electron-main/keyboardLayoutMainService.ts
@@ -5,3 +5,3 @@
 
-import type * as nativeKeymap from 'native-keymap';
+import type * as nativeKeymap from '@vscodium/native-keymap';
 import * as platform from '../../../base/common/platform.js';
@@ -48,3 +48,3 @@ export class KeyboardLayoutMainService extends Disposable implements INativeKeyb
 	private async _doInitialize(): Promise<void> {
-		const nativeKeymapMod = await import('native-keymap');
+		const nativeKeymapMod = await import('@vscodium/native-keymap');
 
